app/assets/stylesheets/config/_colors.scss

```diff
@@ -93,6 +93,11 @@
   --header-icons-color: var(--base-70);
   --header-icons-color-hover: var(--base-100);
 
+  // Global notice
+  --global-notice-bg: var(--accent-success-lighter);
+  --global-notice-text: #000;
+  --global-notice-link: #000;
+
```

Comment:
The entire `global-notice` element is still based on legacy code and UI from pre-Crayons era.

Instead of adding individual styles here and (and also updating ones in shared.scss) I'd much rather update this part https://github.com/forem/forem/blob/master/app/views/layouts/application.html.erb#L8-L10 to use one of our existing pattern, in particular

```
<div class="crayons-banner">
    <%= flash[:global_notice] %>
</div>
```

^ this will:

1. be more accessible since it uses colors with better contrast,
2. be more consistent with similar "banners" we use elsewhere,
3. help keep our frontend codebase up-to-date with Crayons.